recipes/yamlprocessor/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
The license file is already included in the source distribution so you can remove it from here.